.github/workflows/phylogenetic.yaml

```diff
@@ -0,0 +1,110 @@
+name: Phylogenetic
+
+defaults:
+  run:
+    # This is the same as GitHub Action's `bash` keyword as of 20 June 2023:
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
+    #
+    # Completely spelling it out here so that GitHub can't change it out from under us
+    # and we don't have to refer to the docs to know the expected behavior.
+    shell: bash --noprofile --norc -eo pipefail {0}
+
+on:
+  workflow_call:
+    inputs:
+      image:
+        description: 'Specific container image to use for phylogenetic workflow (will override the default of "nextstrain build")'
+        required: false
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      image:
+        description: 'Specific container image to use for ingest workflow (will override the default of "nextstrain build")'
+        required: false
+        type: string
+      trial_name:
+        description: |
+          Trial name for deploying builds.
+          If not set, builds will overwrite existing builds at s3://nextstrain-data/zika*
+          If set, builds will be deployed to s3://nextstrain-staging/zika_trials_<trial_name>_*
+        required: false
+        type: string
+      sequences_url:
+        description: |
+          URL for a sequences.fasta.zst file.
+          If not provided, will use default sequences_url from phylogenetic/defaults/config_zika.yaml
+        required: false
+        type: string
+      metadata_url:
+        description: |
+          URL for a metadata.tsv.zst file.
+          If not provided, will use default metadata_url from phylogenetic/defaults/config_zika.yaml
+        required: false
+        type: string
+
+jobs:
+  set_config_overrides:
+    runs-on: ubuntu-latest
+    steps:
+      - id: config
+        name: Set config overrides
+        env:
+          TRIAL_NAME: ${{ inputs.trial_name }}
+          SEQUENCES_URL: ${{ inputs.sequences_url }}
+          METADATA_URL: ${{ inputs.metadata_url }}
+        run: |
+          config=""
+          if [[ "$TRIAL_NAME" || "$SEQUENCES_URL" || "$METADATA_URL" ]]; then
+
+            config+="--config"
```

Comment:
I wondered if it's possible to avoid this long condition by using `--config` multiple times, but seems like it only takes the last value as we've discovered recently in Augur. Alternatively, `--config` can be added last using a simpler condition:

```sh
config=""

if [[ "$TRIAL_NAME" ]]; then
  config+=" deploy_url='s3://nextstrain-staging/zika_trials_"$TRIAL_NAME"_'"
fi

if [[ "$SEQUENCES_URL" ]]; then
  config+=" sequences_url='"$SEQUENCES_URL"'"
fi

if [[ "$METADATA_URL" ]]; then
  config+=" metadata_url='"$METADATA_URL"'"
fi

if [[ $config ]]; then
  config="--config $config"
fi
```

Not saying that's any better - I think what you have is just fine

Reply:
Ah, thanks for the suggestion! That looks much cleaner!

Reply:
Updated in https://github.com/nextstrain/zika/pull/55/commits/267e72d19ea6d0b7418941901e12bf3c3c72b637